src/util/simplify_expr_int.cpp

```diff
@@ -604,43 +604,42 @@ bool simplify_exprt::simplify_plus(exprt &expr)
 
 bool simplify_exprt::simplify_minus(exprt &expr)
 {
+  auto const &minus_expr = to_minus_expr(expr);
```

Comment:
The lines after this one still use `expr`, not `minus_expr`

Reply:
I've made the functions use them where it seemed reasonable.